rpmio/rpmpgp.c

```diff
@@ -1095,6 +1095,10 @@ static int pgpVerifySelf(pgpDigParams key, pgpDigParams selfsig,
     return rc;
 }
 
+enum {
+    RPM_MAX_OPENPGP_BYTES = 1 << 16,
+};
```

Comment:
Ehm. Why on earth an *enum* of all things?

Reply:
It’s a way to have a compile-time constant in C without a preprocessor macro.  I’ll use a `#define` instead.

Reply:
There's also the good old static const int. 
Also, `1 << 16` makes it look like a bit in a bitfield, which it isn't. Since it's a size, use good old numbers please.